tests/inclusive_scan.cpp

```diff
@@ -48,10 +48,10 @@ void inclusive_scan_gold(std::vector<T>& v, T init = 0,
 
 // test the gold computation against a known value
 TEST_F(InclusiveScanAlgorithm, TestSTDInclusiveScan) {
-  std::vector<int> v = {5, 1, 6, 2, 6, 2, 5, 7};
+  std::vector<int> v    = {5, 1,  6,  2,  6,  2,  5,  7};
   std::vector<int> gold = {5, 6, 12, 14, 20, 22, 27, 34};
 
-  inclusive_scan_gold(v, 0, [](int a, int b) { return a + b; });
+  inclusive_scan_gold(v, 0, std::plus<int>());
```

Comment:
The use of these operators make the code shorter. Nice!
Probably in modern C++ std::plus<int>{}